libfreerdp/core/fastpath.c

```diff
@@ -466,7 +483,7 @@ static int fastpath_recv_update_data(rdpFastPath* fastpath, wStream* s)
 			if ((fastpath->fragmentation != FASTPATH_FRAGMENT_FIRST) &&
 					(fastpath->fragmentation != FASTPATH_FRAGMENT_NEXT))
 			{
-				WLog_ERR(TAG,  "Unexpected FASTPATH_FRAGMENT_LAST");
+				WLog_ERR(TAG, "fastpath_recv_update_data: Unexpected FASTPATH_FRAGMENT_LAST");
```

Comment:
Ok.
I will fix it late, in next pull request, by adding:
#define WLOG_ERR(_tag, _fmt, args...)					\
	    WLog_ERR(_tag, __FUNCTION__": "_fmt, ##args)
and same for WLog_DBG.